### ISSUE TITLE
ST2 Error in console, although package control works

### ISSUE THREAD

**Issue by aziz** [wbond/package_control#870]
```
Reloading plugin ~/Library/Application Support/Sublime Text 2/Packages/0_package_control_loader/02-bz2.py
Traceback (most recent call last):
  File "./sublime_plugin.py", line 62, in reload_plugin
  File "./02-bz2.py", line 2, in <module>
    from package_control import sys_path
ImportError: No module named package_control
```

**Comment by wbond** [comment]
Can you list the contents of `~/Library/Application Support/Sublime Text 2/Packages/0_package_control_loader`?

**Comment by aziz** [comment]
```
ls "$HOME/Library/Application Support/Sublime Text 2/Packages/0_package_control_loader"
02-bz2.py                
02-bz2.pyc               
dependency-metadata.json
```

**Comment by wbond** [comment]
Ah, interesting. There should be a file named `00-package_control.py`. If you remove the folders `0_package_control_loader` and `bz2`, Package Control should regenerate it. I'll add a check to make sure that file is automatically created if missing.

**Comment by aziz** [comment]
Worked like a charm. Thanks.

**Comment by wbond** [comment]
This should be automatically fixed in the next release by the code in be92f50804c62e747585ec6b025e0e07acbed830.
